### PR TITLE
Add timestamp of when the alarm was trigger to slack notifications

### PR DESCRIPTION
**Pull request by alicefuzier**
### What is this PR trying to achieve?
Give us more information when troubleshooting slack errors
### Who is this change for?
Devs
### Have the following been considered/are they needed?

- [ ] Run `terraform apply`.

### CODE REVIEW

File: lambdas/post_to_slack/test_post_to_slack.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                                  'MessageAttributes': {}}}]}
+
+    post_to_slack.main(event, None)
+
+    calls = mock_post.call_args_list
+    print(calls)
+
+    assert len(calls) == 1
+
+    assert calls[0][0][0] == url
```
Comment (alicefuzier) [inline lambdas/post_to_slack/test_post_to_slack.py]: WTF??

Comment (kenoir) [inline lambdas/post_to_slack/test_post_to_slack.py]: 😂

**Comment by kenoir** [comment]
`flake8` is probably upset about whitespace ...

File: lambdas/post_to_slack/test_post_to_slack.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        }]
+    }
+
+    post_to_slack.main(event, None)
+
+    calls = mock_post.call_args_list
+
+    assert len(calls) == 1
+
+    assert calls[0][0][0] == url
```
Comment (alicefuzier) [inline lambdas/post_to_slack/test_post_to_slack.py]: @alexwlchan is there a better way of doing this?

**Pull request merged**